games/k.yaml

```
@@ -78,6 +78,25 @@
   - https://raw.githubusercontent.com/sandsmark/kart/master/startscreen.png
   - https://raw.githubusercontent.com/sandsmark/kart/master/screenshot.png
 
+- name: 'Katawa Shoujo: Re-Engineered'
+  originals:
+  - Katawa Shoujo
+  type: remake
+  repo: 'https://codeberg.org/fhs/katawa-shoujo-re-engineered'
+  url: 'https://www.fhs.sh/projects/ksre'
+  feed: 'https://codeberg.org/fhs/katawa-shoujo-re-engineered/releases.atom'
+  development: very active
+  status: playable
+  content: open
```

```suggestion
  content: free
```
the license is ND/NC which are not open source